js_modules/dagit/packages/core/src/launchpad/ConfigEditorConfigPicker.tsx

```diff
@@ -206,6 +216,27 @@ const ConfigEditorPartitionPicker: React.FC<ConfigEditorPartitionPickerProps> =
       rightElement,
     };
 
+    const {isDynamicPartition, partitionDefinitionName} = React.useMemo(() => {
+      const assetNodes = data?.assetNodes;
+      const firstAssetNodeDefinition = assetNodes?.[0].partitionDefinition;
+      if (
```

Comment:
Hey hey! Hmmm this is a good question.

To open the launchpad from the materialize modal and reach this code, the user has to go through `stateForLaunchingAssets`, so we've already checked that the partitioned assets in the selection share the same partitioning.

I think it'd be best to have this code grab the first partition definition it finds rather than the partitionDefinition of assetNodes[0], because you are allowed to launch some partitioned and some un-partitioned assets at the same time.

I think saying this would do the trick: 

```const definition = assetNodes.find(a => !!a.partitionDefinition)?.partitionDefinition```